### ISSUE TITLE
Wrong argument order in clause for assoc predicate map_assoc/2

### ISSUE THREAD

**Issue by pmoura** [vscosta/yap-6.3#45]
For the following clause:

https://github.com/vscosta/yap-6.3/blob/61adeeeb5c8d9c93786528e8d1f4bbd4652839a7/library/assoc.yap#L216

the Logtalk linter reports:

```
!     Type error: expected variable but got t
!       in clause for predicate map_assoc/2
!       in file /Users/pmoura/Documents/Prolog/yap-6.3/library/assoc.yap between lines 216-217
```

Given the meta-predicate directive for this predicate, it seems the arguments are reversed and the correct clause should be:

```logtalk
map_assoc(_, t) :- !. 
```

**Comment by vscosta** [comment]
thanks again!

**Comment by pmoura** [comment]
Fixed in e894209836a5dd4a7c54ae0d3e2b8a29e0673345.
